cafebabel/templates/home.html

```diff
@@ -39,4 +39,24 @@ <h2>Recently published</h2>
     {{ card(article) }}
   {% endfor %}
 {% endif %}
+  <div id=under-construction>
+  <button></button>
+    <h2>Welcome to the brand new Cafébabel!</h2>
+    <p>
+      We're still under construction.<img src={{ url_for('static', filename='img/emoji/emoji-hammer-40.jpg') }} alt="hammer emoji"><br>
+      But if you want to take a look <img src={{ url_for('static', filename='img/emoji/emoji-eyes-40.jpg') }} alt="eyes emoji"> around
+      then feel free to login and send us your feedback here:<br>
+      <a href=mailto:webmaster@cafebabel.com>webmaster@cafebabel.com</a>
+    </p>
+    <p>
+      The Cafébabel team <img src={{ url_for('static', filename='img/emoji/emoji-smile-40.jpg') }} alt="smile emoji">
+    </p>
+  </div>
+  <script>
```

Comment:
I don't know if it was a good idea, but I put the script just here so that it's easier to delete everything at the same time when I'll do it.